src/azure-cli/azure/cli/command_modules/rdbms/custom.py

```diff
@@ -87,7 +140,39 @@ def _server_create(cmd, client, resource_group_name, server_name, sku_name, no_w
             location=location,
             tags=tags)
 
-    return client.create(resource_group_name, server_name, parameters)
+    server_result = resolve_poller(
+        client.create(resource_group_name, server_name, parameters), cmd.cli_ctx,
+        '{} Server Create'.format(engine_name))
+    user = server_result.administrator_login
+    server_id = server_result.id
+    loc = server_result.location
+    version = server_result.version
+    sku = server_result.sku.name
+    host = server_result.fully_qualified_domain_name
+
+    # Adding firewall rule
+    if public_network_access is not None and start_ip != '':
+        firewall_id = create_firewall_rule(cmd, resource_group_name, server_name, start_ip, end_ip, engine_name)
+
+    logger.warning('Make a note of your password. If you forget, you would have to '
+                   'reset your password with \'az %s flexible-server update -n %s -g %s -p <new-password>\'.',
+                   engine_name, server_name, resource_group_name)
+
+    update_local_contexts(cmd, server_name, resource_group_name, location, user)
+
+    if engine_name == 'postgres':
+        return form_response(user, sku, loc, server_id, host, version,
```

Comment:
response transform could be done with transform as shown in 

https://github.com/Azure/azure-cli/blob/7c03a5d292538f9e8ec923f8b01ce8ed473dbb39/src/azure-cli/azure/cli/command_modules/storage/commands.py#L327

Reply:
table_transform is used to transform for table output. transform is used to format result(response), which is applied to all output format and table_transform also could utilize transformed result.

Reply:
I noticed that the other examples where transformers are leveraged is when the command functions returns the result of an API call directly. In our case, after the API call , there are a number of other operations that need to be completed based on the result of the API call before returning the result ( L: 146 - 161 ) in custom.py. Hence, I was thinking if it is okay to keep the current implementation unless there is an efficient way to do it even by moving it to the transformer. Thoughts ?